brands/amenity/fuel.json

```diff
@@ -3340,5 +3340,14 @@
       "brand:wikipedia": "en:Murco Petroleum",
       "name": "Murco"
     }
+  },
+  "amenity/fuel|QuickChek": {
+    "countryCodes": ["us"],
+    "tags": {
+      "brand": "QuickChek",
```

Comment:
It does need an `"amenity": "fuel"` tag in here too..

Reply:
Okay, Here you go. ^^